.travis.yml

```diff
@@ -1,6 +1,8 @@
 language: python
 python: 2.7
-install: "pip install flake8"
-script: "flake8 ."
+install:
+  - pip install flake8
+script:
+  - flake8 .
```

Comment:
We're not disagreeing, but Travis should just run flake8 and not the actual UI tests at this time.